src/messages/GE_fr-FR.json

```diff
@@ -319,5 +319,6 @@
   "Administrator permission": "L'administration du locataire doit d'abord obtenir une autorisation avant que vous puissiez donner votre consentement",
   "Adaptive Cards designer": "Concepteur de cartes adaptatives.",
   "Click Run Query button above to fetch Adaptive Card": "Cliquez sur le bouton Exécuter la requête ci-dessus pour aller chercher la carte adaptative.",
-  "Your history includes queries made in the last 30 days": "Votre historique inclut les requêtes effectuées au cours des 30 derniers jours"
+  "Your history includes queries made in the last 30 days": "Votre historique inclut les requêtes effectuées au cours des 30 derniers jours",
```

Comment:
@thewahome should this file be edited unless it comes through the loc team?  Just want to make sure it doesn't cause any issues for their tooling. It may be better to have this file removed which they'll localize with everything else?

Reply:
@ddyett  I am also not sure whether it would cause any issues. Now that you mention it, I agree that it would be safer to just remove it

Reply:
Anything I should be doing here or you guys are in control here? 😃 Let me know!

Reply:
You can remove this line on the `GE_fr-FR.json` file

Reply:
This line appeared when merging the latest changes to this branch. It's not directly related to the change. Will this cause impact?